packages/flutter/android/build.gradle

```diff
@@ -4,6 +4,7 @@ group 'com.example.flutter_breez_liquid'
 version '0.2.1'
 
 buildscript {
+    ext.kotlin_version = '1.8.20'
```

Comment:
If there are no obstacles, I'd suggest we upgrade to Kotlin 2.0.0.

Reply:
I would prefer to keep this and the android bindings at the same kotlin version. If need be we can address it in a separate PR